### PR TITLE
Prettyify scene uwu

### PR DESCRIPTION
**Pull request by vjnambi**

### CODE REVIEW

File: src/app/components/explanation-view/explanation-view.component.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
-<h1>Cultural Humility Explanation</h1>
-
-<mat-vertical-stepper>
+<body>
+<div class = "isolated">
+<h1 style= "font-size: 60px">Cultural Humility Explanation</h1>
```
Comment (jeannie3) [inline src/app/components/explanation-view/explanation-view.component.html]: We can move the style here into the scss file

File: src/app/components/scenario-scene-context/scenario-scene-context.component.html [hunk 2]
```diff
@@ -1,5 +1,4 @@
 <h1> {{ scene.title }} </h1>
 
 <p> {{ scene.description}} </p>
-
-<p>{{ scene.type }} </p> 
+<!--<img *ngIf="scene.image_link !== null" [src]="scene.image_link" alt="This is the image that is displayed in the scene." width="150" height="150">-->
```
Comment (sgundlapalli11) [inline src/app/components/scenario-scene-context/scenario-scene-context.component.html]: we can delete this

File: src/app/components/role-list-view/role-list-view.component.html [hunk 3]
```diff
@@ -1,7 +1,13 @@
-<h1>{{scenarioTitle}}</h1>
-<p>{{scenarioDescription}}</p>
-<div *ngFor="let role of formRoles.controls" style="display:inline-block">
+<body>
+<div class ="isolated">
+<h1 style="font-size: 60px">{{scenarioTitle}}</h1>
+</div>
+<p class="professional">{{scenarioDescription}}</p>
+<div class = "isolatedBlueBox">
+<div class = "isolatedWhite" *ngFor="let role of formRoles.controls" style="display:inline-block">
```
Comment (jeannie3) [inline src/app/components/role-list-view/role-list-view.component.html]: The styles here can also be moved to the scss file. Same for the h1 above

File: src/app/components/scenario-scene-context/scenario-scene-context.component.html (hunk 2 above)
Comment (jeannie3) [inline src/app/components/scenario-scene-context/scenario-scene-context.component.html]: This comment could be removed

**Comment by sgundlapalli11** [comment]
Move any style tags in HTML into the css files

File: src/app/components/scene-container/scene-container.component.html [hunk 4]
```diff
@@ -1,9 +1,13 @@
-<mat-grid-list cols="3">
-  <mat-grid-tile colspan="1" rowspan="1" [style.backgroundColor]="'lightblue'">
-    <app-scenario-scene-context [scene]="currentScene"></app-scenario-scene-context>
-    <img *ngIf="currentScene.image_link !== null" [src]="currentScene.image_link" alt="This is the image that is displayed in the scene." width="75" height="75">
-  </mat-grid-tile>
-  <mat-grid-tile colSpan="1" rowSpan="1" [style.backgroundColor]="'lightgreen'">
-    <app-scene-answer-choice-list [answerChoices]="answerChoices" (answered)="updateScene($event)"></app-scene-answer-choice-list>
-  </mat-grid-tile> 
-</mat-grid-list>
+<div class="hflex-container" style="background-color: skyblue; margin: auto; height: 100%">
```
Comment (jeannie3) [inline src/app/components/scene-container/scene-container.component.html]: Same for this file: the styles could be moved to a different folder

File: src/app/login/login.component.html [hunk 5]
```diff
@@ -1,25 +1,50 @@
...
+    </mat-toolbar-header>
+    <mat-card-content>
+            <form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
+                <table class="example-full-width" cellspacing="0">
+                    <tr>
+                        <td>
+                            <mat-form-field>
+                                <div class="form-group">
+                                        <input matInput placeholder="Username" type="text" formControlName="username" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.username.errors }" />
+                                        <!-- <div *ngIf="submitted && f.username.errors" class="invalid-feedback">
```
Comment (jeannie3) [inline src/app/login/login.component.html]: This block of comment could be removed

File: src/app/login/login.component.html [hunk 6]
```diff
@@ -1,25 +1,50 @@
...
+                                </div>
+                            </mat-form-field>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <mat-form-field>
+                                <div class="form-group">
+                                    <input matInput placeholder="Password" type="password" formControlName="password" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.password.errors }" />
+                                    <!-- <div *ngIf="submitted && f.password.errors" class="invalid-feedback">
```
Comment (jeannie3) [inline src/app/login/login.component.html]: This comment could also be removed

File: src/app/register/register.component.html [hunk 7]
```diff
@@ -1,33 +1,65 @@
...
+        </mat-toolbar-header>
+        <mat-card-content>
+            <form [formGroup]="registerForm" (ngSubmit)="onSubmit()">
+                <table class="example-full-width" cellspacing="0">
+                    <tr>
+                        <td>
+                            <mat-form-field>
+                                <div class="form-group">
+                                    <input matInput placeholder="Name" type="text" formControlName="firstName" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.firstName.errors }" />
+                                    <!-- <div *ngIf="submitted && f.firstName.errors" class="invalid-feedback">
```
Comment (jeannie3) [inline src/app/register/register.component.html]: Remove comment

File: src/app/register/register.component.html [hunk 8]
```diff
@@ -1,33 +1,65 @@
...
+                            </mat-form-field>
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td>
+                            <mat-form-field>
+                                    <div class="form-group">
+                                            <input matInput placeholder="Password" type="password" formControlName="password" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.password.errors }" />
+                                            <!-- <div *ngIf="submitted && f.password.errors" class="invalid-feedback">
```
Comment (jeannie3) [inline src/app/register/register.component.html]: comment

File: src/app/components/explanation-view/explanation-view.component.scss [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+        border: 0;
+        outline: 0;
+        font-size: 100%;
+        vertical-align: baseline;
+        background: transparent;
+    }
+
+.centeredBlueBox {margin: auto; border: 3px solid black; border-radius: 5px; padding: 10px; background-color: skyblue; width: 300px}
+
+.isolatedWhite {margin: 10px; text-align: center; background-color: white}
```
Comment (jeannie3) [inline src/app/components/explanation-view/explanation-view.component.scss]: This doesnt seem to be used, so we can remove it

**Comment by jeannie3** [comment]
I think it's ready to be merge. There are only really small unused code that we could remove to make our code cleaner for sure, but other than that, I think it's good.

**Comment by jeannie3** [comment]
> I didn't run the application to check the actual UI design so codewise, this is workable. I'm not sure what the design of the UI we've agreed on so can't really review based on that.
> Good job!

We don't have actual mockups yet. Sunjith and Marcus should be working on it, but as of now, this PR has the temporary fix for our styles.

**Pull request merged**